### PR TITLE
Fix offset cutoff

### PR DESCRIPTION
**Pull request by norkans7**

### CODE REVIEW

File: temba/api/v2/views.py [hunk 1]
```diff
@@ -1,9 +1,15 @@
 from __future__ import absolute_import, unicode_literals
 
+import urlparse
```
Comment (rowanseymour) [inline temba/api/v2/views.py]: I'm always open to discussion on our code style, but I've been consistently separating `import` lines from `from x import y` lines for improved readability.

File: temba/api/v2/views.py [hunk 2]
```diff
@@ -80,6 +86,37 @@ class CreatedOnCursorPagination(pagination.CursorPagination):
 class ModifiedOnCursorPagination(pagination.CursorPagination):
     ordering = '-modified_on'
 
+    def decode_cursor(self, request):
```
Comment (rowanseymour) [inline temba/api/v2/views.py]: I think this should be overridden in a base class of both `ModifiedOnCursorPagination` and `MsgCursorPagination`

File: temba/utils/__init__.py [hunk 3]
```diff
@@ -492,3 +492,15 @@ def print_max_mem_usage(msg=None):
     print "=" * 80
     print msg + locale.format("%d", resource.getrusage(resource.RUSAGE_SELF).ru_maxrss, grouping=True)
     print "=" * 80
+
+
+def positive_int(integer_string, strict=False, cutoff=None):
```
Comment (rowanseymour) [inline temba/utils/__init__.py]: Why not just import the existing `_positive_int` method from DRF ?

File: temba/api/v2/views.py (hunk 2 above)
Comment (rowanseymour) [inline temba/api/v2/views.py]: Could also put that class in `api/support.py`

**Comment by nicpottier** [comment]
yep

**Pull request merged**